### ISSUE TITLE
load more btn appears even when there is nothing to load

### ISSUE THREAD

**Issue by OneShot-Niko** [mybearworld/roarer#27]
![kép](https://github.com/mybearworld/roarer/assets/150537842/0945300f-bfa0-4807-8996-34b8f09b035f)
![kép](https://github.com/mybearworld/roarer/assets/150537842/d1ad9670-4e5f-4015-a408-da5d317a2029)
You need an alt or another person if you wanna test this. First one is at the very start of DMs and the second is before anyone even sends a DM to each other.

Not sure if I should report as bug but decided I'll open up an issue anyways

**Comment by mybearworld** [comment]
Good catch, thanks! There's a `page_amount` value, so I can probably do something with that.
